lib/prawn-rails/document.rb

```diff
@@ -1,6 +1,9 @@
-require 'prawn'
-require 'prawn/table'
-require "prawn-rails/extension"
```

Comment:
Re-add require "prawn-rails/extension" after this loop and remove from inside the loop